frontend/src/components/post/Post.vue

```diff
@@ -50,9 +50,21 @@ export default {
 			default: false
 		}
 	},
+	data() {
+		return {
+			postData: this.post,
+		}
+	},
 	components: {
 		Vote
 	},
+	watch: {
+    	post: {
+        	handler(newValue, oldValue) {
+        		this.postData = newValue;
+        	}
+      	}
+    },
```

Comment:
Where are you using this post watch function?

Reply:
@HunterJS-bit

Reply:
@mittalyashu Because when I created `postData` will not be reactive when prop changes, it will not assign new value from `post` prop when it changes